packages/suite/src/components/suite/HiddenPlaceholder.tsx

```diff
@@ -22,24 +22,42 @@ const Wrapper = styled.span<WrapperProps>`
 `;
 
 interface HiddenPlaceholderProps {
-    intensity?: number;
+    enforceIntensity?: number;
     children: ReactNode;
     className?: string;
     ['data-test']?: string;
 }
 
 export const HiddenPlaceholder = ({
     children,
-    intensity = 5,
+    enforceIntensity,
     className,
     ...rest
 }: HiddenPlaceholderProps) => {
+    const ref = useRef<HTMLSpanElement>(null);
+    const [automaticIntensity, setAutomaticIntensity] = useState(10);
     const discreetMode = useSelector(selectIsDiscreteModeActive);
+
+    useEffect(() => {
+        if (ref.current === null) {
+            return;
+        }
+
+        const fontSize = Number(
+            window
+                .getComputedStyle(ref.current, null)
+                .getPropertyValue('font-size')
+                .replace('px', ''),
+        );
+        setAutomaticIntensity(fontSize / 5);
```

Comment:
honestly, not sure if this is a good idea, since it's not really automatic in the end of the day :c In the future those values can be just hard-coded in accordance with the font-sizes we are using, since it's not so linear, I would assume.

Reply:
I have been playing with it and it seems its working on the scale from normal text to h1 size. Thus I would assume event if its not linear, the non-linearity problem is out of our use-case.

Reply:
I see, so it’s forced only for the graph, my bad. I read the TransactionSummary.tsx file name and missed that it’s a graph wrapper